### PR TITLE
travis ci: build only mate-submodule

### PR DESCRIPTION
**Pull request by raveit65**
This will reduce build time and we are independent from another package.
I am not sure about all the debian stuff under build_scripts:

### CODE REVIEW

File: .build.yml [hunk 1]
```diff
@@ -90,32 +72,14 @@ variables:
     -enable-checker security.insecureAPI.strcpy"'
 
 before_scripts:
```
Comment (sc0w) [inline .build.yml]: seems this line `"before_scripts:"` can be removed too

Comment (raveit65) [reply .build.yml]: It doesn't hurt but OK :)

**Comment by raveit65** [comment]
Looks we need to enable github pages for clang result?

**Review by sc0w — APPROVED** [review]

**Comment by sc0w** [comment]
all the job is done

@yetist please, can you enable https://mate-submodules.mate-desktop.dev ?

File: configure.ac [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+AC_PROG_CC
+AM_DISABLE_STATIC
+AC_PROG_LIBTOOL
+PKG_PROG_PKG_CONFIG
+GLIB_GSETTINGS
+
+AC_PATH_PROG(GLIB_GENMARSHAL, glib-genmarshal)
+AC_PATH_PROG(GLIB_MKENUMS, glib-mkenums)
+AC_PATH_PROG(GLIB_COMPILE_RESOURCES, glib-compile-resources)
```
Comment (rbuj) [inline configure.ac:25]: these variables are not used

File: configure.ac [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+dnl ===========================================================================
+
+PKG_CHECK_MODULES(MS,				\
+	glib-2.0 >= $GLIB_REQUIRED		\
+	gthread-2.0						\
+	gio-unix-2.0 >= $GIO_REQUIRED	\
+	gtk+-3.0 >= $GTK_REQUIRED)
+AC_SUBST(MS_CFLAGS)
+AC_SUBST(MS_LIBS)
```
Comment (rbuj) [inline configure.ac:61]: This should have been merged using just one PKG_CHECK_MODULES

File: configure.ac [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+MATE_COMMON_INIT
+MATE_DEBUG_CHECK([no])
+MATE_COMPILE_WARNINGS
+
+AC_PROG_CC
+AM_DISABLE_STATIC
+AC_PROG_LIBTOOL
+PKG_PROG_PKG_CONFIG
+GLIB_GSETTINGS
```
Comment (rbuj) [inline configure.ac:21]: GLIB_GSETTINGS ? Why do you add this macro if there is no GSettings schema XML file?

**Comment by raveit65** [comment]
Why you asking so late ?
Didn't you noticed that this PR is merged?

**Pull request merged**